wire-ios-sync-engine/WireSyncEngine.xcodeproj/project.pbxproj

```diff
@@ -3694,6 +3694,7 @@
 			buildSettings = {
 				GCC_PREFIX_HEADER = "$(SRCROOT)/Source/WireSyncEngine-iOS.pch";
 				INFOPLIST_FILE = "$(SRCROOT)/Resources/WireSyncEngine-ios-Info.plist";
+				MARKETING_VERSION = 1.0;
```

Comment:
suggestion: it seems MARKETING_VERSION and CURRENT_PROJECT_VERSION were inverted.

CURRENT_PROJECT_VERSION=438.0.4

it should not be a big issue, but for now I would keep the same versions as before until we clean all related hotfixes